full/Dockerfile

```diff
@@ -8,6 +8,15 @@ RUN yes | unminimize \
         build-essential \
         htop \
         jq \
+        hub=2.7.0~ds1-1 \
```

Comment:
Just use git

Reply:
see #76

Reply:
Personally, I like `hub`, because it allows me to run `hub pull-request` (something that `git` doesn't directly allow, and that usually requires jumping around various web pages to achieve).

But if we do want to pre-install `hub` in all Gitpod workspaces, it should also work perfectly out-of-the box.

Have you tried to `brew install hub` in a running Gitpod workspace, and did it work well? (I suspect that it requires some extra configuration before it can work, e.g. setting up a GitHub Personal Access Token.)

Reply:
I think `hub` should be replaced with `gh`: https://github.com/cli/cli